src/hooks/useError.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @flow
+ */
+
+import { getIn } from 'immutable';
+import { useSelector } from 'react-redux';
+
+import { ERROR } from '../constants/redux';
+
+const useError = (path :string[]) :?Object => {
```

Comment:
The return type should be `SagaError`, not `Object`. Having said that, I'm having second thoughts about the `SagaError` name. 

🤔

Reply:
Isn't it possible that someone could set and error in the reducer that is technically not a SagaError? Lmk if you want to change banding.

Reply:
It is possible, but that's the problem. The problem with `Object` is that it is effectively the same as `any`, which is not helpful to us. If `useError` returns an actual error type that we've agreed to, you'll get flow errors if you try to access a non-standard field. I think this will force us to be consistent.

I'm ok with `Object` until we define an error type we all agree on.

Reply:
I definitely think that `SagaError` is a good place to start. Maybe it's good to just do it now and then from there, we can decide how to improve upon it?